app/client/src/features/events/EventDetailsCard/EventDetailsCard.tsx

```diff
@@ -31,16 +34,40 @@ interface Props {
 }
 
 export function EventDetailsCard({ fragmentRef }: Props) {
-    const { title, description } = useFragment(EVENT_DETAILS_CARD_FRAGMENT, fragmentRef);
+    const { title, description, startDateTime, endDateTime } = useFragment(EVENT_DETAILS_CARD_FRAGMENT, fragmentRef);
     const classes = useStyles();
 
+    // helper function to get the start and end time to display in the titlecard
+    const getTime = (dateTime: Date) => {
+        var hours = dateTime.getHours();
+        const mins = dateTime.getMinutes();
+        const suffix = hours >= 12 ? 'PM' : 'AM';
+
+        if (hours > 12) { // converts 24-hour to 12-hour
+            hours -= 12;
+        }
+        else if (hours === 0) { // if midnight hour
+            hours = 12;
+        }
+
+        // add a leading 0 to the string if mins is less than 10
+        const minsStr = mins >= 10 ? String(mins) : '0' + String(mins);
+
+        // combine previous vars to get time as a string
+        const time = String(hours) + ':' + minsStr + suffix;
+        return time;
+    }
+
+    const startTime = startDateTime ? getTime(new Date(startDateTime)) : getTime(new Date());
+    const endTime = endDateTime ? getTime(new Date(endDateTime)) : getTime(new Date());
+
```

Comment:
Make sure to take a look at `src/utils/format.ts`.  We use the [date-fns](https://date-fns.org/) library for date formatting.  Try using that library rather than writing your own custom date formatting function.